public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -170,3 +178,33 @@ export function transformRoleTenantPermissions(
     permissionType: getTenantPermissionType(tenantPermission.allowed_actions),
   }));
 }
+
+export function isPrivateTenant(selectedTenant: string | null) {
+  return selectedTenant !== null && selectedTenant?.startsWith(PRIVATE_TENANT);
```

Comment:
`startsWith` can allow a custom tenant with the prefix `__user__`
Any reason for keeping not using equality ?

Reply:
done